easybuild/main.py

```diff
@@ -303,12 +321,16 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
             eb_file = find_easybuild_easyconfig()
             orig_paths.append(eb_file)
 
-    if len(orig_paths) == 1:
+    if not orig_paths:
+        # if no easyconfig files are specified and we are using --from-pr, use current directory as target directory
+        target_path = os.getcwd() if (options.copy_ec and options.from_pr) else None
+    elif len(orig_paths) == 1 and not (options.copy_ec and options.from_pr):
         # if only one easyconfig file is specified, use current directory as target directory
         target_path = os.getcwd()
     elif orig_paths:
         # last path is target when --copy-ec is used, so remove that from the list
-        target_path = orig_paths.pop() if options.copy_ec else None
+        # use the absolute path as some use cases occur in a temporary directory
```

Comment:
In which case are we copying to the target path from a different directory?
I'm guessing that's with `--from-pr` (we should mention it here rather than leave ourselves guessing in the future)

Perhaps that should be considered a bug?